test/test_ark/test_model.py

```diff
@@ -92,6 +92,36 @@ def test_torch_4bits_sym_cpu(self):
 
         shutil.rmtree("./saved", ignore_errors=True)
 
+    def test_torch_4bits_sym_cpu_awq(self):
+        model = AutoModelForCausalLM.from_pretrained(self.model_name, dtype="auto", trust_remote_code=True)
+        tokenizer = AutoTokenizer.from_pretrained(self.model_name, trust_remote_code=True)
+        bits, group_size, sym = 4, 32, False
```

Comment:
All existing tests use group_size=32. Since group_size=128 is more commonly used in practice, we should add a test for it to ensure proper coverage.